internal/engine/live_update_build_and_deployer.go

```diff
@@ -204,14 +206,14 @@ func liveUpdateInfoForStateTree(stateTree liveUpdateStateTree) (liveUpdInfo, err
 	var hotReload bool
 
 	if luInfo := iTarget.LiveUpdateInfo(); !luInfo.Empty() {
-		var skipped []string
-		fileMappings, skipped, err = build.FilesToPathMappings(filesChanged, luInfo.SyncSteps())
+		var pathsMatchingNoSync []string
+		fileMappings, pathsMatchingNoSync, err = build.FilesToPathMappings(filesChanged, luInfo.SyncSteps())
 		if err != nil {
 			return liveUpdInfo{}, err
 		}
-		if len(skipped) > 0 {
+		if len(pathsMatchingNoSync) > 0 {
 			return liveUpdInfo{}, buildcontrol.RedirectToNextBuilderInfof("found file(s) not matching a LiveUpdate sync, so "+
-				"performing a full build. (Files: %s)", strings.Join(skipped, ", "))
+				"performing a full build. (Files: %s)", strings.Join(pathsMatchingNoSync, ", "))
```

Comment:
as long as we're here, have we considered using ospath.FileListDisplayNames to pretty up this file list?

Reply:
will do!